fundingmanager.go

```diff
@@ -1246,6 +1246,8 @@ func (f *fundingManager) announceChannel(localIDKey, remoteIDKey, localFundingKe
 	f.cfg.SendAnnouncement(ann.chanAnn)
 	f.cfg.SendAnnouncement(ann.chanUpdateAnn)
 	f.cfg.SendAnnouncement(ann.chanProof)
+
+	// TODO: Send node announcement?
```

Comment:
After reflecting, I think a node announcement should indeed follow any local announcements we send out.